app/assets/translations/appOfflineData/bebbo/countryData.ts

```diff
@@ -0,0 +1,541 @@
+export const countryData: any = {
+    "en": [
+        {
+            "CountryID": "6",

```

Comment:
Be consistent throughout the file. Follow the pattern on the left attributes. For example:
1. Use camel case: countryId, countryEmail etc.
or 
2. Use underscore with all lowercase: country_id, country_email etc.

Reply:
@Akhror this data we got from CMS

Reply:
Ah Ok then.